src/input/keyboard/mod.rs

```diff
@@ -751,14 +768,16 @@ impl<'a, D: SeatHandler + 'static> KeyboardInnerHandle<'a, D> {
                     .inner
                     .pressed_keys
                     .iter()
-                    .map(|keycode| {
-                        KeysymHandle {
-                            // Offset the keycode by 8, as the evdev XKB rules reflect X's
-                            // broken keycode system, which starts at 8.
-                            keycode: keycode + 8,
-                            state: &self.inner.state,
-                            keymap: &self.inner.keymap,
-                        }
+                    .filter_map(|keycode| {
+                        MODIFIER_KEYSYMS.contains(keycode).then(|| {
+                            KeysymHandle {
+                                // Offset the keycode by 8, as the evdev XKB rules reflect X's
+                                // broken keycode system, which starts at 8.
+                                keycode: keycode + 8,
+                                state: &self.inner.state,
+                                keymap: &self.inner.keymap,
+                            }
+                        })
```

Comment:
Have you tested this? I am not sure we actually need to forward any pressed keys. The `focus.modifiers` event should be enough to update modifiers on the client-side, I believe.

Reply:
I have! I've been using it with just passing a `Vec::new()` to the `focus.enter` call and it has been working fine that way as well.

I can update my PR to just do that instead if you'd prefer.

Reply:
Yes please, then we don't need that list of modifier keysyms, which I would prefer. :)

Reply:
Agreed! I searched through xkb for a function that would tell me if a Keysym is a modifier but didn't find anything during a quick search.